mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
-// There are syntax errors in this code - can you fix it to pass the tests?
+// There are syntax errors in this code - can you fix it to pass the tests?//
 
-function addNumbers(a b c) {
+function addNumbers(a ,b ,c) {
```

Comment:
This looks good!
:question: Have you tried running prettier on your code? The formatting on these commas is a little inconsistent with the rest of the file